apps/ui/src/components/Ui/InputColor.vue

```diff
@@ -22,10 +22,8 @@ const dirty = ref(false);
 
 const inputValue = computed({
   get() {
-    const defaultValue = props.definition.default;
-
-    if (!model.value && !dirty.value && defaultValue) {
-      return typeof defaultValue === 'function' ? defaultValue() : defaultValue;
+    if (!model.value && !dirty.value && props.definition.default) {
+      return props.definition.default;
```

Comment:
Is there some case we send `default`? i get `undefined` everytime
<img width="391" alt="image" src="https://github.com/user-attachments/assets/f098a71c-3493-4971-bc33-575aaefa4249" />

Reply:
Just reverting a change done here https://github.com/snapshot-labs/sx-monorepo/pull/1179/files#diff-7444314e7d803af7b83e72063a2658917d99ca467cffa9e63bb9efd873f2d5caL23